app/models/operation.rb

```diff
@@ -0,0 +1,8 @@
+class Operation < ApplicationRecord
+  belongs_to :account
+
+  validates :amount, presence: true, numericality: { greater_than: 0 }
+  validates :kind, presence: true
+
+  enum kind: %i[debit credit]
```

Comment:
_Minor suggestion:_ Not needed at first, but in the future we may need an index on `kind`.